<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dg_Setting_add</class>
 <widget class="QDialog" name="Dg_Setting_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wg_check_yq" native="true">
     <widget class="QLabel" name="pic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>869</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> 
</string>
      </property>
      <property name="text">
       <string>  截距设置---卡号为：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>811</x>
        <y>11</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/home/home/guanbi.png);
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MainButton" name="bt_save_setting">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="MainButton" name="bt_canal_setting">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_card">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>16</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>00000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>截距公式：y = k*x + b ; x为原始浓度值</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_setting_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 214, 214);</string>
       </property>
       <property name="text">
        <string>项目1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>93</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>k = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>145</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b = </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prok1_setting">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>79</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prob1_setting">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>130</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_setting_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 214, 214);</string>
       </property>
       <property name="text">
        <string>项目2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prob2_setting">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>130</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>145</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>93</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>k = </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prok2_setting">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>79</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_setting_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>140</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 214, 214);</string>
       </property>
       <property name="text">
        <string>项目3</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prob3_setting">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>131</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>146</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>94</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>k = </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ed_prok3_setting">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>批次号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_pnum_setting">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <zorder>bt_save_setting</zorder>
     <zorder>bt_canal_setting</zorder>
     <zorder>pic</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>l_card</zorder>
     <zorder>label_11</zorder>
     <zorder>frame_setting_1</zorder>
     <zorder>frame_setting_2</zorder>
     <zorder>frame_setting_3</zorder>
     <zorder>label_10</zorder>
     <zorder>ed_pnum_setting</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header location="global">mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
